File: tourneyAPI/Models/Entities/Characters/Lucas.cs

namespace Entities;

using Enums;


// Defines the competitive profile metadata for this playable character.
public class Lucas : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Lucas()
    {
        Id = CharacterId.Lucas;
        characterName = CharacterName.LUCAS;
        archetype = Archetype.MIX_UP;
        fallSpeed = FallSpeed.FLOATY;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.B;
    }

}

File: tourneyClient/src/models/entities/Characters/Lucas.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Lucas: Character =
{
    id: CharacterId.Lucas, 
    characterName: CharacterName.LUCAS,
    archetype: Archetype.MIX_UP,
    fallSpeed: FallSpeed.FLOATY,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.B
}

export default Lucas; 